iota/filters.py

```diff
@@ -69,10 +68,12 @@ class Trytes(f.BaseFilter):
   """
   Validates a sequence as a sequence of trytes.
   """
-  CODE_NOT_TRYTES   = 'not_trytes'
-  CODE_WRONG_FORMAT = 'wrong_format'
+  ADDRESS_BAD_CHECKSUM   = 'address_bad_checksum'
```

Comment:
❔ Should this guy live in the `AddressNoChecksum` class instead of `Trytes`?

💡 We don't have to worry about `templates` getting overwritten; the metaclass will combine them for us automatically:
https://github.com/eflglobal/filters/blob/1.3.2/filters/base.py#L42-L48

Reply:
Oh thank you for the lesson, I didn't know the metaclass combines them. Will do.